### PR TITLE
Pool provisioning config

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR contains a copy of the pre-provisioning setup for each 1ES pool that we operate. This PR represents a proposal for some initial optimizations around our pool sizes. Basically I'm saying that after 9pm each day we scale the pool down to zero pre-allocated and then increase the pre-allocated in the morning up to its current level.

The long term goal for putting this information under version control is that we can make changes and review them, and then automatically apply the configuration to our pools.

NOTE: The times are in UTC.

### CODE REVIEW

File: config/1es-pools/azsdk-pool-mms-win-2019-general/provisioning.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "00:00": 0
+    },
+    {
+        "16:00": 25
```
Comment (weshaggard) [inline config/1es-pools/azsdk-pool-mms-win-2019-general/provisioning.json:6]: Is there a reason to choose 25? As opposed to something like 10?

Comment (mitchdenny) [reply config/1es-pools/azsdk-pool-mms-win-2019-general/provisioning.json:6]: I'm happy to choose 10. The balancing act here is the latency on builds starting. If there are agents available for a job, then you'll see it start in 0-30 seconds most of the time. If not you have to wait 300 seconds for an agent to spin up. If you get too large pipelines come in at the same time then your buffer is mostly exhausted and someone would notice queuing.

If they came in further apart you might not notice. So its a tuning thing.

**Review by weshaggard — APPROVED** [review]
I don't have any real suggestions here. I'm fine starting here and then using the data to make adjustments as patterns emerge.

**Pull request merged**